### DISCUSSION TITLE
Auth scheme deleted

### DISCUSSION THREAD

**Discussion by williamhmw** [supabase/supabase#30274] [Questions] [answered]
Hello, as in some cases I researched, when I was doing a migration through Prisma, just deleting the entire auth schema table, I would like some help from the community to try to restore, I'm a beginner and I'm not able to recover.

I tried to copy the tables from another project, but I still can't create more users.

**Comment by GaryAustin1** [comment, marked as answer]
It would be better to start with a new instance.
If you don't get everything exactly correct with all the grants/rls etc. you may find you have issues much later.

**Comment by williamhmw** [comment]
I'll have to go down this path anyway, thanks for the help anyway!
